ecmascript/minifier/src/option/mod.rs

```diff
@@ -52,6 +52,13 @@ pub struct MangleOptions {
     pub safari10: bool,
 }
 
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ManglePropertiesOptions {
+    #[serde(default, alias = "reserved")]
+    pub reserved: Vec<String>,
```

Comment:
`JsWord` is a reference-counted, interned version of `String`.
Unlike `String`, `JsWord` does not allow modifying content.

It's much faster to compare - one equality comparison of u64 for JsWord but memcmp for `String` - and it's cheaper to clone.
Also, small string, which is very common in ast (because of identifiers) is stored as inline if you use `JsWord`.

Reply:
(But `String` is fast enough in most cases)